src/tir/pass/unroll_loop.cc

```diff
@@ -160,7 +160,9 @@ class LoopUnroller : public StmtExprMutator {
     PrimExpr extent = tir::Simplify(op->extent);
     const IntImmNode  *v1 = extent.as<IntImmNode>();
     int value = -1;
-    if (v1 != nullptr) {
+    // integers that do not fit in int32_t are treated as symbolic,
+    // as it's impossible to unroll such large loops
```

Comment:
should we also print a warning here?

Reply:
I'm not sure. A warning here may be raised for loops that are not marked as `unroll`. So every loop with a i64 extent raises the warning here, even when the user does not intend to unroll it.